src/style.css

```
@@ -0,0 +1,123 @@
+* {
+  box-sizing: border-box;
+}
+
+html {
+  font-family: "Roboto", sans-serif;
+}
+
+body, h2 {
+  margin: 0;
+}
+
+.product-card {
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  width: 200px;
+  border: 1px solid #f3f3f3;
+  border-radius: 5px;
+  padding: 32px 16px 16px;
+  background-color: #fff;
+}
+
+.product-card__poster {
+  width: 160px;
+  height: 134px;
+  margin-bottom: 40px;
+}
+
+.product-card__name {
+  font-size: 12px;
+  font-weight: 500;
+  line-height: 18px;
+  margin-bottom: 4px;
+}
+
+.product-card__code {
+  align-self: flex-start;
+  font-size: 10px;
+  line-height: 14px;
+  font-weight: 400;
+  margin-bottom: 15px;
+  color: #616070;
+}
+
+.product-card__rating {
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  width: 100%;
+  margin-bottom: 24px;
+  color: #060b35;
+}
+
+.stars {
+  display: flex;
+}
+
+.star {
+  background-image: url(images/star.svg);
+  background-position: 50%;
+  background-repeat: no-repeat;
+  width: 16px;
+  height: 16px;
+  margin-right: 4px;
+}
+
+.stars--1 .star:nth-child(1),
+.stars--2 .star:nth-child(-n+2),
+.stars--3 .star:nth-child(-n+3),
+.stars--4 .star:nth-child(-n+4),
+.stars--5 .star:nth-child(-n+5) {
+  background-image: url(images/star-active.svg);
+}
+
+.product-card__reviews {
+  font-size: 10px;
+  line-height: 14px;
+}
+
+.product-card__price {
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  width: 100%;
+  margin-bottom: 16px;
+}
+
+.product-card__price-word {
+  font-size: 12px;
+  font-weight: 400;
+  line-height: 18px;
+  color: #616070;
+}
+
+.product-card__price-value {
+  font-size: 16px;
+  font-weight: 700;
+  line-height: 18px;
+}
```

Color как и в заголовке
```suggestion
.product-card__price-value {
  font-size: 16px;
  font-weight: 700;
  line-height: 18px;
  color: #060B35;
}
```

---

agree